src/main/webapp/play.html

```diff
@@ -86,7 +88,19 @@
 	   %>
 	   var  playOrder = ["webrtc"];
 	   <%
-	}	
+	}
+	
+	 String playType = request.getParameter("playType") ;
+	 if (playType == null || playType.equals("live")) {
+	   %>
+	   var  playType = ["live"];
```

Comment:
I think it will be better if we add a new option(vod) for playOrder.
Then it will try to play mp4 file or webm file.